jupyterhub/vendor_images.yaml

```diff
@@ -1,3 +1,6 @@
 docker.io:
   images-by-tag-regex:
-    bitnami/jupyterhub: ^v?[0-9]+\.[0-9]+?\.[0-9]+$
+    bitnami/jupyterhub: ^v?[0-9]+\.[0-9]+?\.[0-9]+(.*)?$
+    bitnami/jupyter-base-notebook: ^v?[0-9]+\.[0-9]+?\.[0-9]+(.*)?$
+    bitnami/configurable-http-proxy: ^v?[0-9]+\.[0-9]+?\.[0-9]+(.*)?$
+    bitnami/bitnami-shell: ^\d+(-.*)?$
```

Comment:
We'll need to limit these regexes so they only include the current latest version we are deploying and any newer versions. You can look in the bootstrap app for examples of how to create a regex that start at a particular semver.